packages/permissions-provider/test/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`As a developer, I can use \`IfAllowed\` component for conditional rendering based user permissions. 1`] = `"Allowed"`;
+
+exports[`As a developer, I can use \`IfAllowed\` component for pass permission check result via render props. 1`] = `"Allowed = true"`;
+
+exports[`As a developer, I can use \`isAllowed\` for check access via context. 1`] = `"Allowed = false"`;
```

Comment:
Maybe use there an inline snapshot? What do you think?

Reply:
good idea, fixed